doc/reference/orm.rst

```diff
@@ -124,9 +153,117 @@ for partners and one for countries::
         country = partner.country_id        # first pass prefetches all partners
         countries.add(country.name)         # first pass prefetches all countries
 
-Set operations
+.. _reference/orm/models:
+.. _reference/orm/model:
+
+Models
+======
+
+.. autoclass:: odoo.models.BaseModel()
+
+    .. autoattribute:: _auto
+    .. autoattribute:: _register
+
+    .. autoattribute:: _name
+    .. autoattribute:: _description
+
+    .. autoattribute:: _inherit
+    .. autoattribute:: _inherits
+
+    .. autoattribute:: _rec_name
+    .. autoattribute:: _order
+
+    .. autoattribute:: _date_name
+    .. autoattribute:: _fold_name
+
+    .. autoattribute:: _translate
+
+AbstractModel
+-------------
+
+.. autoclass:: odoo.models.AbstractModel
+
+Model
+-----
+
+.. autoclass:: odoo.models.Model
+
+TransientModel
 --------------
 
+.. autoclass:: odoo.models.TransientModel
+
+.. _reference/orm/models/crud:
+
+Common ORM Methods
+==================
+
+.. currentmodule:: odoo.models
+
+Create/update
+-------------
+
+.. automethod:: Model.create
+
+.. automethod:: Model.copy
+
+.. automethod:: Model.default_get
+
+.. automethod:: Model.name_create
+
+.. automethod:: Model.write
+
+Read/search
+-----------
+
+.. automethod:: Model.browse
+
+.. automethod:: Model.name_search
+
+.. automethod:: Model.read_group
+
+.. automethod:: Model.search
+
+.. automethod:: Model.search_count
+
+Fields/Views
```

Comment:
inconsistent underline

Reply:
those methods are read/search on fields/views so Fields/Views is a subsection of Read/Search :D